spec/duration.html

```diff
@@ -781,7 +780,9 @@ <h1>ToPartialDuration ( _temporalDurationLike_ )</h1>
           1. Let _value_ be ? Get(_temporalDurationLike_, _property_).
           1. If _value_ is not *undefined*, then
             1. Set _any_ to *true*.
-            1. Set _value_ to ? ToIntegerOrInfinity(_value_).
+            1. Set _value_ to ? ToNumber(_value_).
+            1. If floor(ℝ(_value_)) ≠ ℝ(_value_), then
```

Comment:
this doesn't account for negative numbers. however, this does:
```suggestion
            1. If ! IsIntegralNumber(_value_) is *false*, then
```

Reply:
Thanks for catching this. I'd better make this change in ToTemporalDurationRecord as well, then.